File: src/main/java/com/learnershi/rclasssocket/entity/controller/HttpController.java
package com.learnershi.rclasssocket.entity.controller;

import com.learnershi.rclasssocket.entity.ClassRoom;
import com.learnershi.rclasssocket.entity.enums.UserType;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RestController
public class HttpController {

    /**
     * 해당 user의 classRoom List를 조회한다.
     *
     * @param userSeq userSeq
     * @param userType userType
     * @return classRoom List
     */
    @RequestMapping("/classRoomList/{userSeq}/{userType}")
    public List<ClassRoom> getClassRoomList(
            @PathVariable String userSeq,
            @PathVariable UserType userType) {
        return null;
    }

}

File: src/main/java/com/learnershi/rclasssocket/controller/HttpController.kt
package com.learnershi.rclasssocket.controller

import com.learnershi.rclasssocket.entity.ClassRoom
import com.learnershi.rclasssocket.entity.enums.UserType
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.PathVariable
import org.springframework.web.bind.annotation.RestController

@RestController
class HttpController(
) {
    /**
     * 해당 user의 classRoom List를 조회한다.
     *
     * @param userSeq userSeq
     * @param userType userType
     * @return classRoom List
     */
    @GetMapping("/classRoomList/{userSeq}/{userType}")
    fun getClassRoomList(
        @PathVariable userSeq: String,
        @PathVariable userType: UserType
    ): List<ClassRoom>? {
        return null
    }
//
//    @PostMapping("/classRoom/{userSeq}/create")
//    fun createClassRoom(
//        @PathVariable userSeq: String,
//        @RequestParam userName: String
//    ): ClassRoom? {
//        return
//    }
}